src/test/java/seedu/anilist/model/ModelManagerTest.java

```diff
@@ -37,14 +37,14 @@ public void setUserPrefs_nullUserPrefs_throwsNullPointerException() {
     @Test
     public void setUserPrefs_validUserPrefs_copiesUserPrefs() {
         UserPrefs userPrefs = new UserPrefs();
-        userPrefs.setAniListFilePath(Paths.get("address/book/file/path"));
+        userPrefs.setAnimeListFilePath(Paths.get("address/book/file/path"));
         userPrefs.setGuiSettings(new GuiSettings(1, 2, 3, 4));
         modelManager.setUserPrefs(userPrefs);
         assertEquals(userPrefs, modelManager.getUserPrefs());
 
         // Modifying userPrefs should not modify modelManager's userPrefs
         UserPrefs oldUserPrefs = new UserPrefs(userPrefs);
-        userPrefs.setAniListFilePath(Paths.get("new/address/book/file/path"));
+        userPrefs.setAnimeListFilePath(Paths.get("new/address/book/file/path"));
```

Comment:
file path should be anime list not address/book.. similarly line 40

Reply:
changed